File: Final_Project/Code/Project_4/Bai_toan_5/Problems/2/CSS_bt5_problem2.py
def is_bipartite_Cn(n):
    # n: so dinh cua do thi Cn
    # Tra ve True neu n chan (Cn luong phan)
    return n % 2 == 0

def is_bipartite_Kn(n):
    # n: so dinh cua do thi Kn
    # Tra ve True neu n <= 2 (Kn luong phan)
    return n <= 2

# Vi du su dung
if __name__ == "__main__":
    print("Is C_4 bipartite?", is_bipartite_Cn(4))  # True
    print("Is K_3 bipartite?", is_bipartite_Kn(3))  # False

File: Final_Project/Code/Project_4/Bai_toan_5/Problems/2/CSS_bt5_problem2.cpp
#include <iostream>
using namespace std;

bool is_bipartite_Cn(int n) {
    // n: so dinh cua do thi Cn
    // Tra ve true neu n chan (Cn luong phan)
    return n % 2 == 0;
}

bool is_bipartite_Kn(int n) {
    // n: so dinh cua do thi Kn
    // Tra ve true neu n <= 2 (Kn luong phan)
    return n <= 2;
}

int main() {
    cout << "Is C_4 bipartite? " << is_bipartite_Cn(4) << endl; // 1
    cout << "Is K_3 bipartite? " << is_bipartite_Kn(3) << endl; // 0
    return 0;
}